app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,9 @@
+module SessionsHelper
+  def current_user?
+    !session[:name].nil?
+  end
+
+  def current_user
+    @current_user ||= User.find(user_id) if (User.id = session[:id])
```

Comment:
This is application logic. There should be no application logic in helpers. (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...)

See [this screenshpt](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from pathwright